src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  $root: &;
+
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background: url("../images/star.svg") no-repeat center;
+    list-style: none;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    @for $index from 1 through 5 {
+      #{$root}--#{$index} &:nth-child(-n + #{$index}) {
+        background: url("../images/star-active.svg") no-repeat center;
```

Comment:
`no-repeat` and `center` properties are not changeable
Delete them from this selector